releasenotes/notes/single-step-instrumentation-support-f40ee4ec8d692c27.yaml

```diff
@@ -0,0 +1,12 @@
+# Each section from every release note are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+features:
+  - |
+    [Beta] Support APM Single Step Instrumentation for Kubernetes.
```

Comment:
Is there a link to public docs that can be linked from here?
Also please include the ways to configure it. You can use this as an example: https://github.com/DataDog/datadog-agent/blob/main/releasenotes/notes/apm-memcached-keep-command-aecd5105a09e1235.yaml

Reply:
There are no public docs yet. This feature is still in private beta , but we hope that by the time the agent will be released it will be public beta.